.swiftlint.yml

```diff
@@ -3,6 +3,7 @@ included:
   - Tests
 disabled_rules:
   - line_length
+  - trailing_comma
```

Comment:
this disables the rule, so no checks are made. To mandate the use of a trailing comma on a multiline collections I think it should read as 
```
trailing_comma:
    mandatory_comma: true
```